### PR TITLE
Subsystem 2

### PR DESCRIPTION
**Pull request by chk1994**
Updated Subsytem2

### CODE REVIEW

File: report.md [hunk 1]
```diff
@@ -54,29 +58,66 @@ The JWT will be:
...
+
+The therapist can then click on the patient's name to view the specific patient's records.
+![therapist read Patient records](https://github.com/IFS4205-2018-Sem1-Team1/design-report/blob/subsystem-2/images/Therapist%20-%20Read%20Bob's%20Records.PNG?raw=true)
+
+- On this page, the therapist would have an overview of all the records of that Patient that the therapist has been given permission to view.
+- The therapist would only have read-only access to the records not owned by the therapist, and write access to records create by the therapist.
+- 'Owner' tab indicates which therapist created that record.
+- To retrieve the files to read the full details of the record, the therapist can click on the filename under the 'Document' tab.
+- The therapist can also use this interface to 'Create' new records and notes, 'Edit' them, and 'Print' these records.
+
```
Comment (chilipadiboy) [inline report.md:93]: Does print mean download in a word file or something?

File: report.md [hunk 2]
```diff
@@ -111,5 +152,24 @@ Notice that this data has 2-anonymity with respect to the attributes `Age`, `Gen
...
 
+### Overview
+
+The other team will be given a special account (which is a type of user, such as `therapist` or `patient`) that can only perform one action: Upload their database to our database. 
+
+### Interface
+
+Upon logging into the special account, the other team will be redirected to a page whereby they can upload their database, image and video files separately. Our current database design is such that the image and video files are stored in a separate file system. The database stores the image and video file names. When a record is retrieved from the database, the server will also retrieve the corresponding image and video files based on their names.
+
+The data can be in tab-separated value (tsv), with the following values:
```
Comment (chilipadiboy) [inline report.md]: Is this a thing?? I though it was still CSV just the delimeter was tab hahaha. Can I clarify what is subtype?

File: report.md [hunk 3]
```diff
@@ -111,5 +152,24 @@ Notice that this data has 2-anonymity with respect to the attributes `Age`, `Gen
...
+1. Type
+1. Subtype
+1. Title
+1. Date_time
+1. Owner_ic
+1. Signature
+1. Content
+
+Where the file names will be stored in `Content`
```
Comment (chilipadiboy) [inline report.md]: Should be randomly generated url links (or at least a string that doesn't make sense) for best practice. For now just keep this :)

File: report.md (hunk 1 above)
Comment (chk1994) [reply report.md:93]: Nope, I am under the impression it is just printing out whatever record the therapist is looking at into hardcopy straight. Cause when you go to a hospital, sometimes you see the nurses ferrying out a trolley full of medical dockets. I was thinking along the line of this kind of print out. Download then print out seems like double effort haha

File: report.md (hunk 2 above)
Comment (chk1994) [reply report.md]: Hmm Im not sure this part came with the branch when I branch from master last night. This is under Subsytem 4 and seem to have been merged into the master already. But I do feel that tsv would be easier to handle here.
From the example given by prof,  type would be the main category(eg. Readings) and subtype would be Blood Pressure(BP) readings

**Pull request merged**